sdk/confidentialledger/Azure.Storage.ConfidentialLedger/api/Azure.Template.netstandard2.0.cs

```diff
@@ -0,0 +1,32 @@
+namespace Azure.Template
```

Comment:
Is this correct?

Reply:
This was  just to see if you were paying attention :)

![](https://media.giphy.com/media/jUwpNzg9IcyrK/giphy.gif)